account/hash.go

```diff
@@ -2,15 +2,15 @@ package account
 
 import (
 	"github.com/NethermindEth/juno/core/felt"
-	starknetgo "github.com/NethermindEth/starknet.go"
+	"github.com/NethermindEth/starknet.go/curve"
 	"github.com/NethermindEth/starknet.go/rpc"
 	"github.com/NethermindEth/starknet.go/utils"
 )
 
 // computeHashOnElementsFelt hashes the array of felts provided as input
 func computeHashOnElementsFelt(feltArr []*felt.Felt) (*felt.Felt, error) {
```

Comment:
We can delete computeHashOnElementsFelt and calculateTransactionHashCommon, and just use the functions define in the hash pkg. 

Also, we can maybe move FmtCalldata directly into accounts.go, and delete the hash file.